File: Backend/Classlib/Model/Vacature.cs
using Newtonsoft.Json;

namespace Classlib
{
    public class Vacature
    {
        public int id { get; set; }

        public int name { get; set; }

        public int description { get; set; }

        public int link { get; set; }

        public int sponsorid { get; set; }
        public Sponsor sponsor { get; set; }
    }
}

File: AndroidApp/app/src/main/java/com/davis/kevin/technicav2/models/Vacature.kt
package com.davis.kevin.technicav2.models

import android.graphics.Bitmap

class Vacature(
    val id: String = "",
    val companyId: String? = null,
    val name: String?  = null,
    val description: String? = null,
    val link:String? = null,
    var image: Bitmap? = null) {
}